tests/openvino/test_quantization.py

```diff
@@ -214,6 +214,9 @@ def preprocess_function(examples, tokenizer):
             # Verify that the configuration is correctly saved and loaded
             loaded_config = OVConfig.from_pretrained(tmp_dir)
             self.assertEqual(ov_config.quantization_config.to_dict(), loaded_config.quantization_config.to_dict())
+            check_optimization_not_applicable_to_optimized_model(
```

Comment:
I see that you've added checks for quantized models when compression is applied to them. Could you please also add this check to some tests which apply compression? For example `test_ovmodel_load_with_compressed_weights` and `test_ovmodel_4bit_auto_compression_with_config`.

Reply:
I now see that check for `test_ovmodel_load_with_compressed_weights` is already added